Makefile

```diff
@@ -115,6 +117,22 @@ workers-generate-cfg:
 		./scripts/generate_workers_cfg.sh $(BACKEND_TYPE) -EFK; \
 	fi
 
+###############
+###  VAULT  ###
+###############
+
+vault-generate-cfg:
+	@$(call displayheader,$(CYAN_COLOR),"${ns} SETUP VAULT")
+	@if [ ! -d "vault/vault/data" ]; then mkdir vault/vault/data; fi
+	@if [ ! -d "vault/vault/logs" ]; then mkdir vault/vault/logs; fi
+	@if [ ! -d "vault/vault/policies" ]; then mkdir vault/vault/policies; fi
+	@echo "CREATE USER ${POSTGRES_USER} WITH PASSWORD '${POSTGRES_PASSWORD}'; ALTER USER ${POSTGRES_USER} WITH SUPERUSER; CREATE DATABASE ${POSTGRES_USER}" > vault/sql/create_user.sql
```

Comment:
Do you think it's the safe way to store credentials in the same database ?
Could it be better to start a new postgres instance ?

Reply:
IDK what it may change from a security perspective, was more concerned about resource consumption..
But if it is safer to have one separate PS for Vault, fine for me